content/en/docs/_index.md

```diff
@@ -14,7 +14,7 @@ menu:
 
 The LumenPnP is an open source Pick and Place project led by [Opulo](https://www.opulo.io/). The LumenPnP is designed to be low cost and suited towards [mid-scale manufacturing](http://stephenhawes.com/level-2-manufacturing/).
 
-{{< container-image path="hero-alpha-min.png" alt="" >}}
+{{< container-image path="images/hero-alpha-min.png" alt="hero image of LumenPnP" >}}
```

Comment:
I just added alt text and the fixed the path now that we're storing that hero image in the shared `assets/images` folder